runelite-client/src/main/java/net/runelite/client/plugins/suppliestracker/SuppliesTrackerPanel.java

```diff
@@ -1,4 +1,4 @@
- /*
+	/*
```

Comment:
Remove this tab and the one in SuppliesTrackerPlugin.

Reply:
Thank you finished